### PR TITLE
Update course.json outline

### PR DESCRIPTION
**Pull request by workshop-maybe**
Team - here is my proposed update to the course.json file

- Changed the order of Modules 201 and 202
- Revised some module titles
- Revised most SLTs in the 200-level Modules

Please take a look + share feedback here.

Thanks!

### CODE REVIEW

**Comment by jinglescode** [comment]
have questions about 101 and 102. 

- feels like `Mesh Design Patterns` is not a good title
- thinking to break out 101.2 101.3 101.4 into `Wallet`
- wondering if 102.1 and 102.2 can be grouped
- wondering if 102.3 should be 101.1

File: src/andamio/data/course.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
             "slts": [
                 {
                     "id": "102.1",
-                    "slt": "I can use a Mesh Provider to get information about the blockchain."
+                    "slt": "I can use a Mesh Providers to get information about the blockchain."
```
Comment (HinsonSIDAN) [inline src/andamio/data/course.json]: Does that mean fetcher providers like maestro or blockfrost? If so, the word `Mesh Providers` looks a bit confusing, coz we use the `MeshProvider` to wrap the app.ts.

Suggest change to something like
- blockchain indexing Providers
- external Providers
- fetcher Providers

Comment (HinsonSIDAN) [reply src/andamio/data/course.json]: or just a Provider from Mesh

**Comment by HinsonSIDAN** [comment]
> * feels like Mesh Design Patterns is not a good title
- 102.1 and 102.2 are called `Utilities` in website, maybe just make it `Mesh Utilities`? If 102.3 is moved to elsewhere

> * wondering if 102.1 and 102.2 can be grouped
- I vote for separating them, as they serve quite different functions. And user can only feel them when they really try to build something. I suspect these lessons are people going quickly at the beginning, but certainly something to revisit

> * wondering if 102.3 should be 101.1
- I think can do

> * thinking to break out 101.2 101.3 101.4 into Wallet
- A bit overkill to put all hooks at the beginning for learners. Perhaps we can have a separate module at the end, as optional, to cover most details?
- If the current plan is to cover all details, grouping is good imo

**Comment by jinglescode** [comment]
@SIDANWhatever @workshop-maybe have made the changes. take a few minutes to look at https://github.com/MeshJS/mesh-pbl/blob/39f1a54e4a68b48fe11f1c360bd73066bad41f8f/src/andamio/data/course.json

**Comment by Zagita21** [comment]
@workshop-maybe Made some minor edits, but generally I really like these changes. I

I agree also mostly with @SIDANWhatever - it is good to keep things in small, neat chunks to avoid overwhelm at the beginning. As James said, there is no problem with repetition for reinforcement - in fact this is good.

**Pull request merged**